SPECS/filesystem/filesystem.spec

```
@@ -704,6 +708,9 @@ return 0
 %config(noreplace) /etc/modprobe.d/tipc.conf
 
 %changelog
+* Thu Sep 14 2022 Thara Gopinath <tgopinath@microsoft.com> - 1.1-12
+- Create missing systemd user and group accounts
```

Small request: let's explicitly list the added accounts, so it's easier to see what changed just by reading the changelog.

```suggestion
- Add the 'systemd-coredump' and 'systemd-oom' user accounts.
```